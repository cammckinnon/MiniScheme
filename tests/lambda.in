((lambda ()
  3))

((lambda (a)
    (a +))
   (lambda (a) (a 2 3)))

((lambda (a b c)
    ((lambda (a) a) 4)
    a)
   7 7 7)

((lambda (a b c d)
    ((a) (b) (c) d))
   (lambda () ((lambda (a) +) 1000))
   (lambda () ((lambda (a b c d) (+ a d)) -2 5000 10000 3))
   (lambda () ((lambda (d) (d)) (lambda () 10)))
   100)