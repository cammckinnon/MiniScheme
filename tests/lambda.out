3.0
5.0
7.0
111.0